### DISCUSSION TITLE
Feature: Calc folders and better management.

### DISCUSSION THREAD

**Discussion by kazad** [kazad/instacalc#64] [Announcements]
A [frequently requested feature](https://github.com/kazad/instacalc/discussions/63), InstaCalc now has:

* Calc folders (including drag and drop)
* Folder open/close status is remembered per-device. If you open/close folders on one computer and come back, it will stay. Each computer has its own state though (so the open/close status won't sync). (I think it'd be annoying to be forced into the same view in mobile, desktop, chrome extension, etc.)
* Better calc management (easier to clone/delete/rename calcs from the list)

<img width="470" alt="Screenshot 2023-11-08 at 9 25 47 AM" src="https://github.com/kazad/instacalc/assets/115572/0095065a-043e-4e61-b5d5-5fb01794c9b5">

This has been on my backlog for a while, glad to get it out!

**Comment by soundstep** [comment]
Wow nice, works beautifully! One small detail in case you want to sort it out, but I'm not even sure it matters. Once a calc is in a folder, I don't think I can drag it back to the root (works only if I drag it to another folder, but really nitpicking).
Thanks a lot for the feature!

**Reply by kazad** [reply]
Thanks! Ah, you can drag it back out, but the UI doesn't highlight the root folder as a target area. I'll update that. Appreciate the suggestion, I'm already using it to clean up my giant list of calcs :).

**Reply by kazad** [reply]
Ok, fixed-up version should be live.

https://github.com/kazad/instacalc/assets/115572/a9dc526b-d4a9-45d8-8648-7217f0cea5d6

**Reply by soundstep** [reply]
Got it, thanks for the fast update!
